src/autocomplete.js

```diff
@@ -137,11 +140,36 @@ class Autocomplete extends Component {
   }
 
   selectItem = itemValue => {
-    this.reset()
+    const previousValue = this.state.selectedValue
+    if (!this.props.multiple) {
+      this.reset()
+    }
     this.setState(
-      {selectedItem: itemValue, value: this.getValue(itemValue)},
+      state => {
+        if (this.props.multiple) {
+          const values = [...state.selectedValue]
+          const pos = values.indexOf(itemValue)
+          if (pos > -1) {
+            values.splice(pos, 1)
+          } else {
+            values.push(itemValue)
+          }
+          return {
+            selectedValue: values,
+            inputValue: values.map(value => this.getValue(value)).join(', '),
```

Comment:
In the future I expect we'll need to make this a prop as well `getInputValue`... Either that or instead of mapping all the values to `this.getValue` we just call `this.getValue(values)`... Seems like that'd be reasonable... Should we do that now?